codyze-console/src/test/kotlin/de/fraunhofer/aisec/codyze/console/ApplicationTest.kt

```diff
@@ -31,40 +31,52 @@ import de.fraunhofer.aisec.cpg.TranslationConfiguration
 import de.fraunhofer.aisec.cpg.TranslationContext
 import de.fraunhofer.aisec.cpg.TranslationManager
 import de.fraunhofer.aisec.cpg.TranslationResult
-import de.fraunhofer.aisec.cpg.graph.Component
-import de.fraunhofer.aisec.cpg.graph.Name
+import de.fraunhofer.aisec.cpg.graph.*
 import de.fraunhofer.aisec.cpg.graph.concepts.arch.Agnostic
+import de.fraunhofer.aisec.cpg.graph.concepts.file.File
 import de.fraunhofer.aisec.cpg.graph.concepts.flows.Main
 import de.fraunhofer.aisec.cpg.graph.declarations.FunctionDeclaration
 import de.fraunhofer.aisec.cpg.graph.declarations.TranslationUnitDeclaration
+import de.fraunhofer.aisec.cpg.graph.statements.expressions.Block
 import de.fraunhofer.aisec.cpg.graph.statements.expressions.CallExpression
-import io.github.detekt.sarif4k.ArtifactLocation
-import io.github.detekt.sarif4k.Location
-import io.github.detekt.sarif4k.Message
-import io.github.detekt.sarif4k.PhysicalLocation
-import io.github.detekt.sarif4k.Region
-import io.github.detekt.sarif4k.Result
-import io.github.detekt.sarif4k.Run
-import io.github.detekt.sarif4k.SarifSchema210
-import io.github.detekt.sarif4k.Tool
-import io.github.detekt.sarif4k.ToolComponent
-import io.github.detekt.sarif4k.Version
+import de.fraunhofer.aisec.cpg.graph.statements.expressions.Reference
+import io.github.detekt.sarif4k.*
 import io.ktor.client.call.*
 import io.ktor.client.plugins.contentnegotiation.*
 import io.ktor.client.request.*
 import io.ktor.client.statement.*
 import io.ktor.http.*
 import io.ktor.serialization.kotlinx.json.*
 import io.ktor.server.testing.*
-import java.io.File
-import kotlin.test.Test
-import kotlin.test.assertContains
-import kotlin.test.assertEquals
-import kotlin.test.assertNotNull
-import kotlin.uuid.Uuid
+import kotlin.test.*
 
 /** A mock configuration for the translation manager. */
-val mockConfig = TranslationConfiguration.builder().sourceLocations(File("some/path")).build()
+val mockConfig =
+    TranslationConfiguration.builder().sourceLocations(java.io.File("some/path")).build()
+
+/** A mock translation unit. */
+val mockTu =
+    TranslationUnitDeclaration().apply {
```

Comment:
Is there a specific reason why you neither use the `NodeBuilder` nor the Fluent-DSL?

Reply:
Sort of. I don't really have a translation context at this stage (and the service does not need one, only the finished nodes), so its hard to invoke them at this point.